docs/loading-aia-and-hmi.rst

```diff
@@ -0,0 +1,58 @@
+.. _loading_aia_and_hmi_files:
+
+========================================================
+Loading and Overplotting AIA and HMI files as SunPy Maps
+========================================================
+
+Loading an AIA file
+-------------------
+To load an AIA file as a SunPy Map with the ``glue-solar`` plugin of ``glue`` is
+very straightforward. Upon firing up ``glue`` while the environment is with
+``glue-solar`` installed, simply import data with the third icon from the left on the top
+navigation bar of ``glue``, choose the file one would like to view, then before clicking
+the "Open" button on the lower right corner of the loader select from the dropdown menu
+at the bottom the "SunPy Map" format. Otherwise the AIA file would be loaded automatically
+as an ordinary FITS file instead. The colormap used in ``glue`` for this AIA file would
+then be one for the AIA format, depending on the wavelength.
+
+Loading a corresponding HMI file
+--------------------------------
+The procedure is similar, except that the default colormap for each of the HMI files
```

Comment:
This makes it sounds like each HMI file has a different colour map.

Reply:
They do in fact...

Reply:
They do?!

Reply:
Yup, otherwise it would be difficult to tell the AIA layer and HMI layer apart, which is crucial for the overplotting.

Reply:
Good point.